src/index.html

```diff
@@ -11,7 +11,179 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="form-field" for="surname">
+          Surname:
+          <input
+            id="surname"
+            type="text"
+            name="surname"
+            required
+          >
+        </label>
+
+        <label class="form-field" for="name">
+          Name:
+          <input
+            id="name"
+            type="text"
+            name="surname"
+            required
+          >
+        </label>
+
+
+
+        <label class="form-field" for="age">
```

Comment:
```suggestion
        </label>

        <label class="form-field" for="age">
```
No need in so many empty lines